services/static-webserver/client/source/class/osparc/ui/message/FlashMessage.js

```diff
@@ -33,11 +33,23 @@ qx.Class.define("osparc.ui.message.FlashMessage", {
     this.base(arguments);
     this._setLayout(new qx.ui.layout.HBox(15));
 
+    const decorator = new qx.ui.decoration.Decorator().set({
+      width: 1,
+      style: "solid",
+      color: this.self().LOG_LEVEL_COLOR_MAP[level].color
+    });
```

Comment:
You can use `services/static-webserver/client/source/class/osparc/theme/Decoration.js` to add decorations